kubetest/README.md

```diff
@@ -32,7 +32,8 @@ Common alternatives:
 ```
 go run hack/e2e.go  # from kubernetes/kubernetes
 go install k8s.io/test-infra/kubetest  # if you check out test-infra
-bazel run //kubetest  # Use bazel to build and run
+bazel run //kubetest  # use bazel to build and run
+go build -o kubetest . # when inside "test-infra/kubetest" directory
```

Comment:
now that https://github.com/kubernetes/test-infra/pull/4732 goes in we wanna ensure kubetest is placed in your $PATH, probably we should remove this.

Reply:
sure! i will remove it.